include/core-image-bsp.inc

```diff
@@ -0,0 +1,31 @@
+# Basic packages/tools for RZ/G2L-SBC
+IMAGE_INSTALL:append = " \
+    kernel-image \
+    kernel-devicetree \
+    bash \
+    coreutils \
+    can-utils \
+    i2c-tools \
+    spitools \
+    python3-smbus \
+    python3-smbus2 \
+    ethtool \
+    iproute2 \
+    iperf3 \
+"
+
+# Install overlays folder to root filesystem
+add_overlays_rootfs() {
+    install -d ${IMAGE_ROOTFS}/boot/overlays
+    cp -r "${DEPLOY_DIR_IMAGE}/target/images/dtbs/overlays" "${IMAGE_ROOTFS}/boot/"
+}
+
+# Install rzpi.dtb to root filesystem
+add_dtb_rootfs() {
```

Comment:
add_overlays_rootfs()
Not all boards use this function to create dtbs overlay.

add_dtb_rootfs()
Hardcoding name of rzsbc dtb.

 Should we add machine suffix for these functions?

Reply:
Updated in the commit: https://github.com/Renesas-SST/meta-renesas/compare/bee65d49d368f5e35779eed10e83f6a9a50ad5a3..de7186edbd510d4722d284e38443ce313d4e89a1#diff-b6efa7a64720e3f1fbb95eff10cf0ee94c5e5711f3075113782f57aba73ce562R32